docs/partials/guides.pug

```diff
@@ -34,7 +34,7 @@
 
     Then you can pass an array of these nodes as the `options` prop.
   p.tip
-    :markdown-it(inline)
+    :markdown-it
```

Comment:
![image](https://user-images.githubusercontent.com/1289884/37141812-3604479e-22f1-11e8-9cd4-bee23d572d84.png)
I got this after removing the `(inline)`...

Reply:
I add to remove this `(inline)` to  make the dev script pass.